src/cincinnati/mod.rs

```diff
@@ -277,7 +277,13 @@ fn evaluate_deadend(node: &Node) -> Option<String> {
     if deadend_reason.is_empty() {
         deadend_reason = "(unknown reason)".to_string();
     }
-
+    if !deadend_reason.is_empty() {
+        let _ = std::fs::remove_file("/etc/motd");
```

Comment:
I think we want to write to e.g. `motd.d/zincati.motd` instead, no?

Reply:
Indeed, this should plug into https://github.com/coreos/console-login-helper-messages.